### DISCUSSION TITLE
Voice Conversion Colab Notebook does not work on A100-SXM4

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#1124] [General]
Hi,

As this is not a bug in YourTTS per se, I move the original issue here, so that it is known to the maintainers but without showing the "bug" tag.

Running the Colab notebook for Voice Conversion on an A100-SXM4 fails when restoring the model because of :

`A100-SXM4-40GB with CUDA capability sm_80 is not compatible with the current PyTorch installation. The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_70.`

The embeddings cannot be computed.

Can PyTorch be upgraded without compatibility / regression concerns ?

**Comment by erogol** [comment]
Hi! 
It's a torch problem. Try updating it. It should work fine.
